base/util.jl

```diff
@@ -256,6 +313,11 @@ for the effects of compilation.
 
 See also [`@time`](@ref), [`@timev`](@ref), [`@timed`](@ref),
 and [`@elapsed`](@ref).
+
+```julia
+julia> @allocated rand(10^6)
```

Comment:
Not sure. Predictable on my computer but perhaps might be OS dependent. That's why I didn't make it into a test.